spec/unit/hanami/router/url_spec.rb

```diff
@@ -3,13 +3,18 @@
 RSpec.describe Hanami::Router do
   let(:router) do
     e = endpoint
+    # rubocop:disable Layout/ExtraSpacing
```

Comment:
Can we remove this and just comply with the rule instead? It's not as clean looking but it's still fine, especially since this is just a spec

Reply:
Sure thing, removed it.